src/net.h

```diff
@@ -682,23 +682,23 @@ class V1TransportDeserializer : public TransportDeserializer
         Reset();
     }
 
-    bool Complete() const
+    bool Complete() const override
     {
         if (!in_data)
             return false;
         return (hdr.nMessageSize == nDataPos);
     }
-    void SetVersion(int nVersionIn)
+    void SetVersion(int nVersionIn) override
```

Comment:
In commit "Add override/final modifiers to V1TransportDeserializer" (ed2dc5e48abed1cde6ab98025dc8212917d47d21)

This function is still never called. Isn't it pointless for CNetMessage and TransportDeserializer to both have SetVersion methods, instead of just setting the version in a single place? See previous https://github.com/bitcoin/bitcoin/pull/16202#discussion_r336638822